### ISSUE TITLE
Error on frappecloud installation

### ISSUE THREAD

**Issue by asoral** [NagariaHussain/doppio_bot#4]
The error that shows up, the moment I send a simple message to the client. The installation is done on frappe cloud

`### App Versions
```
{
	"chat": "0.0.1",
	"doppio_bot": "0.0.1",
	"erpnext": "14.22.3",
	"frappe": "14.33.1",
	"helpdesk": "0.8.1",
	"hrms": "14.2.2",
	"payments": "0.0.1",
	"sales_commission": "0.0.1",
	"surplus_asset_management": "0.0.1"
}
```
### Route
```
doppio-bot
```
### Traceback
```
Traceback (most recent call last):
  File "env/lib/python3.10/site-packages/redis/connection.py", line 559, in connect
    sock = self._connect()
  File "env/lib/python3.10/site-packages/redis/connection.py", line 615, in _connect
    raise err
  File "env/lib/python3.10/site-packages/redis/connection.py", line 603, in _connect
    sock.connect(socket_address)
OSError: [Errno 99] Cannot assign requested address

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "apps/frappe/frappe/app.py", line 66, in application
    response = frappe.api.handle()
  File "apps/frappe/frappe/api.py", line 54, in handle
    return frappe.handler.handle()
  File "apps/frappe/frappe/handler.py", line 45, in handle
    data = execute_cmd(cmd)
  File "apps/frappe/frappe/handler.py", line 83, in execute_cmd
    return frappe.call(method, **frappe.form_dict)
  File "apps/frappe/frappe/__init__.py", line 1607, in call
    return fn(*args, **newargs)
  File "apps/doppio_bot/doppio_bot/api.py", line 43, in get_chatbot_response
    return conversation_chain.run(prompt_message)
  File "env/lib/python3.10/site-packages/langchain/chains/base.py", line 213, in run
    return self(args[0])[self.output_keys[0]]
  File "env/lib/python3.10/site-packages/langchain/chains/base.py", line 106, in __call__
    inputs = self.prep_inputs(inputs)
  File "env/lib/python3.10/site-packages/langchain/chains/base.py", line 193, in prep_inputs
    external_context = self.memory.load_memory_variables(inputs)
  File "env/lib/python3.10/site-packages/langchain/memory/buffer.py", line 39, in load_memory_variables
    return {self.memory_key: self.buffer}
  File "env/lib/python3.10/site-packages/langchain/memory/buffer.py", line 24, in buffer
    self.chat_memory.messages,
  File "env/lib/python3.10/site-packages/langchain/memory/chat_message_histories/redis.py", line 50, in messages
    _items = self.redis_client.lrange(self.key, 0, -1)
  File "env/lib/python3.10/site-packages/redis/client.py", line 1975, in lrange
    return self.execute_command('LRANGE', name, start, end)
  File "env/lib/python3.10/site-packages/redis/client.py", line 898, in execute_command
    conn = self.connection or pool.get_connection(command_name, **options)
  File "env/lib/python3.10/site-packages/redis/connection.py", line 1192, in get_connection
    connection.connect()
  File "env/lib/python3.10/site-packages/redis/connection.py", line 563, in connect
    raise ConnectionError(self._error_message(e))
redis.exceptions.ConnectionError: Error 99 connecting to localhost:6379. Cannot assign requested address.

```
### Request Data
```
{
	"type": "POST",
	"args": {
		"prompt_message": "Can you tell me your name",
		"session_id": "38vGQU8_UO9mqNKbOyD9L"
	},
	"headers": {},
	"error_handlers": {},
	"url": "/api/method/doppio_bot.api.get_chatbot_response"
}
```
### Response Data
```
{
	"exception": "redis.exceptions.ConnectionError: Error 99 connecting to localhost:6379. Cannot assign requested address."
}
````

**Comment by NagariaHussain** [comment]
Looks like redis is not running on the assumes port, for now you can manually pass the redis url on this line:

```

message_history = RedisChatMessageHistory(session_id=session_id)

```

I will figure out a better way in the meantime (for FC).

**Comment by NagariaHussain** [comment]
Fixed with 596d2cec368eebb33a370d09df86c063d9462ed3
